Library/Homebrew/cmd/info.rb

```diff
@@ -298,7 +298,7 @@ def info_formula(formula)
         kegs = [
           *heads.sort_by { |keg| -keg.tab.time.to_i },
           *versioned.sort_by(&:scheme_and_version),
-        ]
+        ].select { |keg| keg.tab.tap == formula.tap }
```

Comment:
> Seems fine though do we want to handle the case where a tab tap is `nil` any differently?

Not sure when that'll happen, but I guess we can do something like this in case it ever does:

```suggestion
        ].select { |keg| (tap = keg.tab.tap).nil? || tap == formula.tap }
```